CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## [Unreleased 0.3.1]
 
+**Notes for self-hosted etcd**:
+Updating to this release requires recreation of existing self-hosted etcd cluster.
```

Comment:
The newly introduced TLS feature for self hosted etcd is a breaking change. Updating...